frontend/src/components/GShootListRowActions.vue

```diff
@@ -52,14 +52,21 @@ SPDX-License-Identifier: Apache-2.0
         />
       </v-list-item>
       <v-divider />
-      <v-list-item>
+      <v-list-item v-if="!canForceDeleteShoot">
         <g-shoot-action-delete-cluster
-
           v-model="deleteClusterDialog"
           :shoot-item="shootItem"
           text
         />
       </v-list-item>
+      <v-list-item v-if="canForceDeleteShoot">
+        <g-shoot-action-force-delete
+          v-model="forceDeleteDialog"
+          :shoot-item="shootItem"
+          button
```

Comment:
do we need this??

Reply:
no not since the recent refactoring. did slip in on merging